apps/meteor/app/api/server/v1/chat.ts

```diff
@@ -325,7 +326,7 @@ API.v1.addRoute(
 			}
 
 			// Permission checks are already done in the updateMessage method, so no need to duplicate them
-			await Meteor.callAsync('updateMessage', { _id: msg._id, msg: this.bodyParams.text, rid: msg.rid });
+			await Meteor.callAsync('updateMessage', { _id: msg._id, msg: this.bodyParams.text, rid: msg.rid }, this.bodyParams.previewUrls);
```

Comment:
we need to get rid of these `Meteor.callAsync` on backend 🙈

Reply:
Done! I moved the Meteor method content to an async function and called it directly there